lib/right_api_client/client.rb

```diff
@@ -112,10 +114,23 @@ class Client
     # @return [Boolean] whether to retry idempotent requests that fail
     attr_reader :enable_retry
 
+    # @return [Hash] hash of RS API 1.5 metadata
+    attr_reader :api_metadata
+
     # Instantiate a new Client, then login if necessary.
     def initialize(args)
       raise 'This API client is only compatible with Ruby 1.8.7 and upwards.' if (RUBY_VERSION < '1.8.7')
 
+      api_json = begin
+        RestClient.get(API_15_METADATA_URL)
+      rescue RestClient::Forbidden => e
+        # Failed to fetch the API 1.5 metadata JSON, so use the most recent
+        # copy stored in this repo
+        puts 'Failed to fetch RS API 1.5 metadata - using local copy.'
```

Comment:
I don't see how it can hurt on the other hand, and I don' think it is going to be a frequent message.  I think we should keep it.